Mathlib/Data/Matrix/Mul.lean

```diff
@@ -879,6 +882,9 @@ theorem vecMul_transpose [Fintype n] (A : Matrix m n α) (x : n → α) : x ᵥ*
   ext
   apply dotProduct_comm
 
+theorem mulVec_eq_sum [Fintype n] (v : n → α) (M : Matrix m n α) : M *ᵥ v = ∑ i, v i • Mᵀ i := by
```

Comment:
Can you write this as
```suggestion
theorem mulVec_eq_sum [Fintype n] (v : n → α) (M : Matrix m n α) : M *ᵥ v = ∑ i, MulOpposite.op (v i) • Mᵀ i := by
```
so that it doesn't need commutativity?

Reply:
I would argue not; for instance, [Matrix.vecMul_ofNat](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Data/Matrix/Mul.html#Matrix.vecMul_ofNat) is not primed, and talks about `MulOpposite`.

You can make the statement look much less awkward with `open scoped RightActions`.

Reply:
I'm specifically unexcited by the idea of needing an extra `rw` within a sigma binder - the infoview appearance won't help with that.

Reply:
There are six other references to `MulOpposite` in this file, and none of them are in primed lemmas. So I really think we should _only_ have the noncommutative version. I don't think `simp_rw [op_smul_eq_smul]` is a particularly high cost to pay, and being presented with these noncommutative versions first makes it easier for downstream lemmas to also work in the non-commutative cases.

Reply:
Ok.